nixos/modules/system/boot/systemd.nix

```diff
@@ -783,19 +784,6 @@ in
 
     services.dbus.enable = true;
 
-    system.activationScripts.systemd = stringAfter [ "groups" ]
-      ''
-        mkdir -m 0755 -p /var/lib/udev
```

Comment:
Same for this one

Reply:
And this was also used in pre-systemd times.